### PR TITLE
Chloesky_ex

### LINKED ISSUE #16661: cholesky_ex

### PR DESCRIPTION
**Pull request by MalikHaroonKhokhar**
Close #16661

### CODE REVIEW

**Comment by MalikHaroonKhokhar** [comment]
- [ ] cholesky_ex #16661

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by MalikHaroonKhokhar** [comment]
@paulaehab

File: ivy/functional/frontends/torch/linalg.py [hunk 1]
```diff
@@ -263,3 +263,15 @@ def solve_ex(A, B, *, left=True, check_errors=False, out=None):
             result = A * math.nan
             info = ivy.ones(A.shape[:-2], dtype=ivy.int32)
             return result, info
+def cholesky_ex(A, *, upper=False, check_errors=False, out=None):
```
Comment (paulaehab) [inline ivy/functional/frontends/torch/linalg.py:266]: Please add the decorator  @to_ivy_arrays_and_back

File: ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py [hunk 2]
```diff
@@ -1289,3 +1289,33 @@ def test_torch_solve_ex(
...
+    x = np.matmul(x.T, x) + np.identity(x.shape[0])  # make symmetric positive-definite
+
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        rtol=1e-01,
+        input=x,
```
Comment (paulaehab) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py:1316]: tha function expect A argument not input argument

**Review by paulaehab — COMMENTED** [review]
Hi @MalikHaroonKhokhar  I am sorry for late response this PR just  slipped under my radar!  please make this changes and make sure that all test run for this function

**Comment by MalikHaroonKhokhar** [comment]
@paulaehab i have made the above changes

**Comment by MalikHaroonKhokhar** [comment]
Hmm more tests failed I will review it tomorrow

**Comment by paulaehab** [comment]
> Hmm more tests failed I will review it tomorrow

No problem , When you finished just request review and I will review it.

**Comment by MalikHaroonKhokhar** [comment]
@paulaehab please review now !

**Comment by MalikHaroonKhokhar** [comment]
@khushi-411 hi khushi you're are new assignee can you review the PR?

**Comment by MalikHaroonKhokhar** [comment]
@khushi-411 idk why these 11 tests are failing here

**Comment by MalikHaroonKhokhar** [comment]
@khushi-411 @paulaehab

**Pull request closed without merging**